applications/ConvectionDiffusionApplication/custom_strategies/strategies/explicit_runge_kutta_4_eulerian_convdiff_strategy.h

```diff
@@ -0,0 +1,385 @@
+        auto& r_process_info = r_model_part.GetProcessInfo();
+        // Call the base method
+        BaseType::Initialize();
+        // If required, initialize the OSS projection variables
+        if (r_process_info[OSS_SWITCH]) {
+            for (auto& r_node : r_model_part.GetCommunicator().LocalMesh().Nodes()) {
+                r_node.SetValue(SCALAR_PROJECTION, 0.0);
+            }
+        }
+        r_process_info.GetValue(TIME_INTEGRATION_THETA) = 1.0;
+
+        KRATOS_CATCH("");
+    }
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    std::string Info() const override
+    {
+        return "ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion";
+    }
+
+    /// Print information about this object.
+    void PrintInfo(std::ostream &rOStream) const override
+    {
+        rOStream << Info();
+    }
+
+    /// Print object's data.
+    void PrintData(std::ostream &rOStream) const override
+    {
+        rOStream << Info();
+    }
+
+    ///@}
+
+protected:
+    ///@name Protected static Member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    /**
+     * @brief Initialize the Runge-Kutta substep.
+     * @details Calculate the orthogonal subscale projections if required.
+     */
+    virtual void InitializeRungeKuttaIntermediateSubStep() override
+    {
+        KRATOS_TRY;
+
+        BaseType::InitializeRungeKuttaIntermediateSubStep();
+
+        auto& r_model_part = BaseType::GetModelPart();
+        auto& r_process_info = r_model_part.GetProcessInfo();
+        // set interpolation parameter TIME_INTEGRATION_THETA
+        if (r_process_info.GetValue(RUNGE_KUTTA_STEP) == 1) {
+            r_process_info.GetValue(TIME_INTEGRATION_THETA) = 1.0;
+        }
+        else if (r_process_info.GetValue(RUNGE_KUTTA_STEP) == 2 || r_process_info.GetValue(RUNGE_KUTTA_STEP) == 3) {
+            r_process_info.GetValue(TIME_INTEGRATION_THETA) = 0.5;
+        }
+        else {
+            KRATOS_ERROR << "RUNGE-KUTTA step not correctly set.";
+        }
+        // execute OSS step, if needed
+        if (r_process_info[OSS_SWITCH] == 1) {
+            ExecuteOSSStep();
+        }
+
+        KRATOS_CATCH("");
+    };
+
+    /**
+     * @brief Initialize the Runge-Kutta substep.
+     * @details Calculate the orthogonal subscale projections if required.
+     */
+    virtual void InitializeRungeKuttaLastSubStep() override
+    {
+        KRATOS_TRY;
+
+        BaseType::InitializeRungeKuttaLastSubStep();
+        auto& r_model_part = BaseType::GetModelPart();
+        auto& r_process_info = r_model_part.GetProcessInfo();
+        // set interpolation parameter TIME_INTEGRATION_THETA
+        if (r_process_info.GetValue(RUNGE_KUTTA_STEP) == 4) {
+            r_process_info.GetValue(TIME_INTEGRATION_THETA) = 0.0;
```

Comment:
Same comment, I'd expect theta to be 1.0 in the last RK step.

Reply:
After talking with you and Riccardo, we agreed to interpolate as follows: `f=theta*f^(n) + (1-theta)*f^(n+1)`. Or at least I understood this.
This is why it needs to be 1 in the first step, and why I initialize to 1. If you prefer the opposite, let me know and I will change.

Reply:
I think the opposite is what one would expect in the explicit case.